packages/server/lib/routes.integration.test.ts

```diff
@@ -1,34 +1,86 @@
 import { migrateLogsMapping } from '@nangohq/logs';
 import { seeders } from '@nangohq/shared';
-import { multipleMigrations } from '@nangohq/database';
+import db, { multipleMigrations } from '@nangohq/database';
+import { migrate as migrateKeystore } from '@nangohq/keystore';
 import { afterAll, beforeAll, describe, expect, it } from 'vitest';
 import { runServer } from './utils/tests.js';
 
 let api: Awaited<ReturnType<typeof runServer>>;
-describe('GET /logs', () => {
+describe('route', () => {
     beforeAll(async () => {
         await multipleMigrations();
         await migrateLogsMapping();
+        await migrateKeystore(db.knex);
 
         api = await runServer();
     });
     afterAll(() => {
         api.server.close();
     });
+    describe('GET /logs', () => {
+        it('should handle invalid json', async () => {
+            const { env } = await seeders.seedAccountEnvAndUser();
+            const res = await fetch(`${api.url}/api/v1/environment/callback`, {
+                method: 'POST',
+                body: 'undefined',
+                headers: { Authorization: `Bearer ${env.secret_key}`, 'content-type': 'application/json' }
+            });
 
-    it('should handle invalid json', async () => {
-        const { env } = await seeders.seedAccountEnvAndUser();
-        const res = await fetch(`${api.url}/api/v1/environment/callback`, {
-            method: 'POST',
-            body: 'undefined',
-            headers: { Authorization: `Bearer ${env.secret_key}`, 'content-type': 'application/json' }
+            expect(await res.json()).toStrictEqual({
+                error: {
+                    code: 'invalid_json',
+                    message: expect.any(String) // unfortunately the message is different depending on the platform
+                }
+            });
         });
+    });
 
-        expect(await res.json()).toStrictEqual({
-            error: {
-                code: 'invalid_json',
-                message: expect.any(String) // unfortunately the message is different depending on the platform
-            }
+    describe('GET /integrations', () => {
```

Comment:
ideally this is enforced at each endpoint in their respective integration tests (when available), also we need to enforce the opposite now